test/network/scenarios/p2p/peers.js

```diff
@@ -72,53 +75,45 @@ module.exports = function(configurations) {
 		});
 
 		describe('forging', () => {
-			before(done => {
-				// Expect some blocks to be forged after 30 seconds
-				const timesToCheckNetworkStatus = 30;
-				let timesNetworkStatusChecked = 0;
-				const checkNetworkStatusInterval = 1000;
-
-				const checkingInterval = setInterval(() => {
-					common.getNodesStatus(params.sockets, (err, data) => {
-						const { networkMaxAvgHeight } = data;
-						timesNetworkStatusChecked += 1;
-						if (err) {
-							clearInterval(checkingInterval);
-							return done(err);
-						}
-						utils.logger.log(
-							`network status: height - ${
-								networkMaxAvgHeight.maxHeight
-							}, average height - ${networkMaxAvgHeight.averageHeight}`
-						);
-						if (timesNetworkStatusChecked === timesToCheckNetworkStatus) {
-							clearInterval(checkingInterval);
-							return done(null, networkMaxAvgHeight);
-						}
+			before(() => {
+				return network.waitForBlocksOnAllNodes(3).then(() => {
+					return new Promise(resolve => {
+						// Add 5 seconds to give time for networkHeight
+						// to update across all nodes.
+						setTimeout(() => {
+							resolve();
+						}, 5000);
 					});
-				}, checkNetworkStatusInterval);
+				});
 			});
 
-			describe('network status after 30 seconds', () => {
-				let getNodesStatusError;
+			describe('network status after 3 blocks', () => {
+				let getAllNodesStatusError;
 				let networkHeight;
 				let networkAverageHeight;
-				let peersCount;
-				let peerStatusList;
+				// TODO: Uncomment when networkHeight issue has been fixed.
```

Comment:
Could you please insert the issue reference here and everywhere where "networkHeight" issue is mentioned?